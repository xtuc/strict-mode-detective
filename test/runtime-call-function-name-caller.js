function f() {
  f.caller;
}

f();
