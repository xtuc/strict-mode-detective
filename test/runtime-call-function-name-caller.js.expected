  1 | function f() {
> 2 |   f.caller;
    |  ^^^^^^^^
  3 | }
  4 | 
  5 | f();
potentially illegal f.caller usage 

